SDWebImage/SDWebImageCompat.m

```diff
@@ -56,10 +56,12 @@
                     scale = 3.0;
                 }
             }
-
-            UIImage *scaledImage = [[UIImage alloc] initWithCGImage:image.CGImage scale:scale orientation:image.imageOrientation];
-            scaledImage.sd_imageFormat = image.sd_imageFormat;
-            image = scaledImage;
+            
+            if (scale != 1) {
```

Comment:
Can we change this to this ? I think it's better than a hard-coded `1`, because user may accidentlly use this method on a retina scaled image.

```
if (scale != image.scale)
```